content/developer/howtos/accounting_localization.rst

```diff
@@ -371,16 +373,17 @@ Functional overview of financial reports is here: :doc:`/applications/finance/ac
 
 Some good examples:
 
-* `l10n_ch_reports/data/account_financial_html_report_data.xml <{GITHUB_ENT_PATH}/l10n_ch_reports/data/account_financial_html_report_data.xml>`_
-* `l10n_be_reports/data/account_financial_html_report_data.xml <{GITHUB_ENT_PATH}/l10n_be_reports/data/account_financial_html_report_data.xml>`_
+* `l10n_ch_reports/data/balance_sheet.xml <{GITHUB_ENT_PATH}/l10n_ch_reports/data/balance_sheet.xml>`_
+* `l10n_be_reports/data/profit_and_loss.xml <{GITHUB_ENT_PATH}/l10n_be_reports/data/profit_and_loss.xml>`_
 
 For the fields' meaning, dive directly to the source:
 
-* `account.financial.html.report (v15) <https://github.com/odoo/enterprise/blob/d4eff9d39469cf3fe18589a1547cb0cdb93f4ae9/account_reports/models/account_financial_report.py#L59-L75>`_
-* `account.financial.html.report.line (v15) <https://github.com/odoo/enterprise/blob/d4eff9d39469cf3fe18589a1547cb0cdb93f4ae9/account_reports/models/account_financial_report.py#L931-L964>`_
+* `account.report (v16) <https://github.com/odoo/enterprise/blob/d4eff9d39469cf3fe18589a1547cb0cdb93f4ae9/account_reports/models/account_financial_report.py#L59-L75>`_
+* `account.report.line (v16) <https://github.com/odoo/enterprise/blob/d4eff9d39469cf3fe18589a1547cb0cdb93f4ae9/account_reports/models/account_financial_report.py#L931-L964>`_
```

Comment:
Changing the version is... weird.

This is wrong, linked to old models not existing anymore
Since the fields are moved to community, you can use the `automodel` directive and link to that